src/items/class-features/dream_sending_(su).json

```diff
@@ -0,0 +1,115 @@
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.udtZxolGt5mMq9SM]{Dreamer}</p>\n<p>Once per day while inside dreamspace, you can briefly communicate with creatures at great distances. This creature can be anywhere in the universe, including on another plane, as long as you’re familiar with them. You can speak freely with any ally linked by your @UUID[Compendium.sfrpg.class-features.ox6Am16XZqBXJNT9]{Telepathic Bond (Sp)} class feature when creating a dream sending. Creatures that aren’t linked to you by a telepathic bond can attempt a Will save with a DC equal to 15 + one-half your class level to resist the dream sending. A creature can choose to fail this save. On a failure, the creature experiences a dream containing a brief message of 25 words or less the next time it rests. The message can be verbal or nonverbal, and you control the aesthetic, imagery, and events of this dream. The creature wakes remembering your message, though you can’t influence a creature’s actions or attitude toward you using this ability.</p>"
+    },
+    "descriptors": {
+      "acid": false,
+      "air": false,
+      "calling": false,
+      "chaotic": false,
+      "charm": false,
+      "cold": false,
+      "compulsion": false,
+      "creation": false,
+      "curse": false,
+      "darkness": false,
+      "death": false,
+      "disease": false,
+      "earth": false,
+      "electricity": false,
+      "emotion": false,
+      "evil": false,
+      "fear": false,
+      "fire": false,
+      "force": false,
+      "good": false,
+      "healing": false,
+      "language-dependent": false,
+      "lawful": false,
+      "light": false,
+      "mind-affecting": false,
+      "pain": false,
+      "poison": false,
+      "polymorph": false,
+      "radiation": false,
+      "scrying": false,
+      "sense-dependent": false,
+      "shadow": false,
+      "sonic": false,
+      "summoning": false,
+      "teleportation": false,
+      "water": false
+    },
+    "duration": {
+      "units": "",
+      "value": null
+    },
+    "formula": "",
+    "isActive": null,
+    "modifiers": [],
+    "range": {
+      "additional": "",
+      "per": "",
+      "units": "",
+      "value": null
+    },
+    "recharge": {
+      "charged": false,
+      "value": null
+    },
+    "requirements": "12th Level (Mystic)",
+    "rollNotes": "",
+    "save": {
+      "type": "",
+      "dc": null,
```

Comment:
Add the DC for this ability

Reply:
I can find no setting in Foundry for how to set that value...Please advise

Reply:
At the very bottom of the details tab, there's a saving throw and skill check field. Put the field there. May require a saving throw/skill check activation type. That is the preferred way to do saving throws, not inline formulas showing DCs